website/components/GoalForm/GoalForm.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <div class="container ">
+    <form action="">
+      <h2 class="text-center">
+        Create a New Goal
+      </h2>
+      <InputField
+        id="task-name"
+        label="Goal Title"
+        type="text"
+        name="task-name"
+        required
+        placeholder="eg: Testing setup for dashboard"
+        autofocus
+      />
+      <InputField
+        id="title"
+        label="Task Name"
+        type="text"
+        name="title"
+        required
+        placeholder="eg: Testing setup for dashboard"
+      />
+      <InputField
+        id="links"
+        label="Links"
+        type="text"
+        name="links"
+        required
+        placeholder="Comma separated links"
+      />
+      <div class="input-container">
+        <SelectOption
+          label="User Role"
+          name="user-role"
+          required
+          :options="usernameCategoryOptions"
+        />
+        <InputField
+          id="username"
+          label="Username"
+          type="text"
+          name="username"
+          required
+          placeholder="eg: sahsisunny"
+        />
+      </div>
+      <div class="input-container">
+        <SelectOption
+          label="Priority"
+          name="priority"
+          required
+          :options="goalPriorityOptions"
+        />
+        <SelectOption
+          label="Type"
+          name="type"
+          required
+          :options="goalTypeOptions"
+        />
+      </div>
+      <div class="input-container">
+        <InputField
+          id="due-date"
+          label="Due Date"
+          type="date"
+          name="due-date"
+          required
+          placeholder="eg: 2020-01-01"
+        />
+        <InputField
+          id="assignee"
+          label="Assignee"
+          type="text"
+          name="assignee"
+          required
+          placeholder="eg: ankush"
+        />
```

Comment:
Can we use the use the `PopupAssignee` component here?

Reply:
added